openscenario/openscenario_interpreter/CMakeLists.txt

```diff
@@ -34,6 +34,20 @@ ament_auto_add_library(${PROJECT_NAME} SHARED
 
 target_link_libraries(${PROJECT_NAME} Boost::filesystem glog pugixml)
 
+target_precompile_headers(${PROJECT_NAME}
+  PUBLIC
+    "<nlohmann/json.hpp>"
+    "<rclcpp/rclcpp.hpp>"
+    "<traffic_simulator/api/api.hpp>"
+    "<pugixml.hpp>"
+
+    "<openscenario_interpreter/expression.hpp>"
+    "<openscenario_interpreter/object.hpp>"
+    "<openscenario_interpreter/pointer.hpp>"
+    "<openscenario_interpreter/record.hpp>"
+    "<openscenario_interpreter/scope.hpp>"
+    "<openscenario_interpreter/simulator_core.hpp>")
```

Comment:
Please sort lexicographically, unless there is a technical reason why declaration order matters. Also, please remove blank lines.